tools/dev-server/index.bzl

```diff
@@ -0,0 +1,125 @@
+load("@build_bazel_rules_nodejs//internal/common:sources_aspect.bzl", "sources_aspect")
+
+"""
```

Comment:
The block comments in this file seem a little awkward and have an extra space after them I think?

Should they be single line for alot of them?

"""Gets the workspace name of the given rule context."""

Reply:
I intentionally put them into their own line (for consistency with other custom rules and to be more readable). The weird extra spacing comes from buildifier and I can't change it 😥

Reply:
Will update to the single-line for comments that fit into the single-line 👍